device/cluster.cpp

```diff
@@ -195,218 +195,295 @@ void Cluster::initialize_interprocess_mutexes(int pci_interface_id, bool cleanup
         return harvested_rows_per_target;
     }
     std::unordered_map<chip_id_t, uint32_t> default_harvesting_masks = {};
-    for(const auto chip : target_devices_in_cluster) default_harvesting_masks.insert({chip, 0});
+    for (const auto chip : target_devices_in_cluster) {
+        default_harvesting_masks.insert({chip, 0});
+    }
     return default_harvesting_masks;
 }
 
-void Cluster::construct_cluster(const std::string& sdesc_path, const uint32_t &num_host_mem_ch_per_mmio_device, const bool skip_driver_allocs,
-                                   const bool clean_system_resources, bool perform_harvesting, std::unordered_map<chip_id_t, uint32_t> simulated_harvesting_masks) {
-    
+void Cluster::construct_cluster(
+    const std::string& sdesc_path,
+    const uint32_t& num_host_mem_ch_per_mmio_device,
+    const bool skip_driver_allocs,
+    const bool clean_system_resources,
+    bool perform_harvesting,
+    std::unordered_map<chip_id_t, uint32_t> simulated_harvesting_masks) {
     std::unordered_set<chip_id_t> target_mmio_device_ids;
-    for (auto &d: target_devices_in_cluster){
-        log_assert(ndesc->get_all_chips().find(d) != ndesc->get_all_chips().end(), "Target device {} not present in current cluster!", d);
-        if (ndesc->is_chip_mmio_capable(d)){
+    for (auto& d : target_devices_in_cluster) {
+        log_assert(
+            ndesc->get_all_chips().find(d) != ndesc->get_all_chips().end(),
+            "Target device {} not present in current cluster!",
+            d);
+        if (ndesc->is_chip_mmio_capable(d)) {
             target_mmio_device_ids.insert(d);
-        }
-        else {
+        } else {
             target_remote_chips.insert(d);
         }
     }
 
-    // It is mandatory for all devices to have these TLBs set aside, as the driver needs them to issue remote reads and writes.
+    // It is mandatory for all devices to have these TLBs set aside, as the driver needs them to issue remote reads and
+    // writes.
     auto architecture_implementation = tt::umd::architecture_implementation::create(arch_name);
-    dynamic_tlb_config["LARGE_READ_TLB"] =  architecture_implementation->get_mem_large_read_tlb();
+    dynamic_tlb_config["LARGE_READ_TLB"] = architecture_implementation->get_mem_large_read_tlb();
     dynamic_tlb_config["LARGE_WRITE_TLB"] = architecture_implementation->get_mem_large_write_tlb();
     dynamic_tlb_config["REG_TLB"] = architecture_implementation->get_reg_tlb();
     dynamic_tlb_config["SMALL_READ_WRITE_TLB"] = architecture_implementation->get_small_read_write_tlb();
 
-    for(const auto& tlb : dynamic_tlb_config) {
-        dynamic_tlb_ordering_modes.insert({tlb.first, TLB_DATA::Relaxed}); // All dynamic TLBs use Relaxed Ordering by default; MT: Good for BH
+    for (const auto& tlb : dynamic_tlb_config) {
+        dynamic_tlb_ordering_modes.insert(
+            {tlb.first, TLB_DATA::Relaxed});  // All dynamic TLBs use Relaxed Ordering by default; MT: Good for BH
```

Comment:
Not related to the change, but I think we can remove last part of the comment stating `MT: Good for BH`. I guess Milos was just clarifying that we can use relaxed for Blackhole, we can delete it now